packages/stockflux-launcher/src/reducers/child-window/Actions.js

```diff
@@ -0,0 +1,8 @@
+export default {
```

Comment:
I'd be inclined to make these all uppercase with underscore separators since this effectively functions like an enum. Just personal preference though - if there's an existing convention then I'd stick to that. Not something I think should stop this PR though,